### PR TITLE
flatpak: add info and remove examples

### PR DESCRIPTION
**Pull request by evaleries**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/flatpak.md [hunk 1]
```diff
@@ -25,3 +25,12 @@
 - List all configured remote sources:
 
 `flatpak remote-list`
+
+- Remove an installed application:
+
+`flatpak remove {{name/application id}}`
```
Comment (bl-ue) [inline pages/linux/flatpak.md:31]: I think we could be a bit more consisten with the other examples here.
```suggestion
`flatpak remove {{name}}`
```

File: pages/linux/flatpak.md [hunk 2]
```diff
@@ -25,3 +25,12 @@
...
+
+- Get an info from installed application:
+
+`flatpak info {{name/application id}}`
```
Comment (bl-ue) [inline pages/linux/flatpak.md:35]: Same here.
```suggestion
`flatpak info {{name}}`
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Thank you for the page update @evaleries (and filling it up to the max of 8 examples :wink:).  I've left two comments below for you to review.

File: pages/linux/flatpak.md [hunk 3]
```diff
@@ -25,3 +25,12 @@
...
+
```
Comment (bl-ue) [inline pages/linux/flatpak.md:36]: Apply this suggestion to fix @tldr-bot's warning.
```suggestion
```

**Comment by bl-ue** [comment]
Also @evaleries, will you please sign the [CLA](https://cla-assistant.io/tldr-pages/tldr?pullRequest=5422) to pass the `license/cla` CI check?

**Review by bl-ue — APPROVED** [review]
Very nice, thank you @evaleries!!

File: pages/linux/flatpak.md [hunk 4]
```diff
@@ -25,3 +25,11 @@
 - List all configured remote sources:
 
 `flatpak remote-list`
+
+- Remove an installed application:
+
+`flatpak remove {{name}}`
+
+- Get an info from installed application:
```
Comment (marchersimon) [inline pages/linux/flatpak.md:33]: Just a little suggestion to keep it consistent with our other pages:
```suggestion
- Show information about an installed application:
```

**Review by owenvoke — APPROVED** [review]
LGTM after @marchersimon's comment has been addressed.

File: pages/linux/flatpak.md (hunk 4 above)
Comment (marchersimon) [reply pages/linux/flatpak.md:33]: @evaleries why did you mark that as resolved?

**Review by bl-ue — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]

**Comment by bl-ue** [comment]
Welcome to tldr-pages, @evaleries! 🎉

**Pull request merged**